### PR TITLE
created rank enum

### PR DESCRIPTION
**Pull request by E-tyedye22**
Here is the rank, from Ace to King

### CODE REVIEW

File: Rank.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    FIVE("Five"),
+    SIX("Six"),
+    SEVEN("Seven"),
+    EIGHT("Eight"),
+    NINE("Nine"),
+    TEN("Ten"),
+    JACK("Jack"),
+    QUEEN("Queen"),
+    KING("King");
+
```
Comment (bloomkd46) [inline Rank.java:15]: Do we want to add a Joker?

**Review by pauldalyiii — APPROVED** [review]
good! I can use this format for my card class. it should return "Ace of Spades" etc.

**Review by bloomkd46 — COMMENTED** [review]
It looks okay, I just have 1 question:

File: Rank.java (hunk 1 above)
Comment (E-tyedye22) [reply Rank.java:15]: Maybe not because aren't there only 1 or 2 jokers, not 4

Comment (bloomkd46) [reply Rank.java:15]: @pauldalyii Do you have any preference?

Comment (bloomkd46) [inline Rank.java:15]:
```suggestion
    KING("King"),
    JOKER(“Joker”);

```

**Review by bloomkd46 — APPROVED** [review]

**Pull request merged**